packages/url-parser-universal/src/index.ts

```diff
@@ -1,4 +1,4 @@
-import { HttpEndpoint, QueryParameterBag, UrlParser } from "@aws-sdk/types";
+import { Endpoint, QueryParameterBag, UrlParser } from "@aws-sdk/types";
```

Comment:
nit: unused imports
```suggestion
import { UrlParser } from "@aws-sdk/types";
```

Reply:
Resolved in https://github.com/aws/aws-sdk-js-v3/pull/404/commits/4387c11f42651b3c5222af1a6fc8ae999e7b58db